app/forms/did_challenge_hmrc_form.rb

```diff
@@ -0,0 +1,3 @@
+class DidChallengeHmrcForm < BaseForm
```

Comment:
@digitalronin and I had a discussion about this on Friday, basically the intention is to keep the name of the thing fairly close to the question we are asking, in this case "_Did you challenge HMRC?_".

I think that conveys the intention of the form object more, considering all our form objects are questions – but happy to have more of a debate around this!